requirements/base.txt

```diff
@@ -15,6 +15,7 @@ djangorestframework-simplejwt==4.4.0
 django-elasticsearch-dsl==7.1.0
 django-elasticsearch-dsl-drf==0.20.5
 
+bs4==0.0.1
```

Comment:
this seems weird? BeautifulSoup is on version 4.8.2.
https://pypi.org/project/beautifulsoup4/

Reply:
I had it at 4.8.2, but it broke-- said there was no bs4 4.8.2. I think whatever abstraction of BeautifulSoup bs4 is, is on version 0.0.1.  Are you able to include bs4 using 4.8.2?

Reply:
Switched it!